linode/rdns/framework_resource_schema.go

```diff
@@ -23,21 +23,24 @@ var frameworkResourceSchema = schema.Schema{
 		"rdns": schema.StringAttribute{
 			Description: "The reverse DNS assigned to this address. For public IPv4 addresses, this will be set " +
 				"to a default value provided by Linode if not explicitly set.",
-			Required: true,
+			Optional: true,
 			Validators: []validator.String{
 				stringvalidator.LengthBetween(3, 254),
 			},
 		},
+		"reserved": schema.BoolAttribute{
+			Description: "Whether the IP address is reserved.",
+			Optional:    true,
+		},
```

Comment:
I've given this change a bit of thought and I think it might be worth excluding the `reserved` field from the `linode_rdns` resource since it seems a tiny bit out of scope and can be managed directly through the IP resources. 

I'd be interested to hear everyone's input on this!